sdk/communication/azure-communication-administration/README.md

```diff
@@ -39,47 +39,47 @@ via the endpoint() and httpClient() functions respectively.
 
 <!-- embedme ./src/samples/java/com/azure/communication/administration/ReadmeSamples.java#L37-L48 -->
 ```java
-// You can find your endpoint and access key from your resource in the Azure Portal
```

Comment:
Not sure why this was removed from the samples?

Reply:
Fixed there was an unsed impor removed from the sample files that broke almost everything